### ISSUE TITLE
Indented bulleted list produces erroneous gap

### ISSUE THREAD

**Issue by jdkruzr** [pbek/QOwnNotes#1504]
![image](https://user-images.githubusercontent.com/618198/70744731-4eeb6580-1cf0-11ea-9dd7-b2cd1dd54758.png)

Note the gap between the third to last and second to last line, as rendered. This is on MacOS Catalina.

**Comment by pbek** [comment]
Yes, that's a known issue of the `QTextBrowser`, like #437 or #1117.

**Comment by pbek** [comment]
Should be fixed with #3466.

<img width="1779" height="688" alt="Image" src="https://github.com/user-attachments/assets/37eceff2-83b9-44b2-842b-9db5ce99e4ce" />
